docs/advanced_topics/images/renditions.md

```diff
@@ -129,9 +149,11 @@ The following `AbstractImage` model methods are involved in finding and generati
 
     .. automethod:: get_rendition
 
-    .. automethod:: find_existing_rendition
```

Comment:
Did you intend to remove `find_existing_rendition` from the documentation here?

Reply:
Yes. Now that we don't actually use it as part of rendition generation, we should probably think about removing it at some point in the near future. Maybe officially deprecating it instead is the better solution?

Reply:
@gasman In that case, would you mind if we leave things for now and take care of the deprecation later on? There is no like-for-like replacement for it at this time, so further work may be required.

Reply:
Sure, let's keep it then.

Reply:
Documentation restored in 23a59e9